crates/aptos-runtimes/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "aptos-runtimes"
```

Comment:
Yeah, good question. I think we can leave this for future work -- i.e., there's a bunch of these crates we could merge. The problem is that it's not super obvious because it's really easy to create cyclic dependencies in our code. So, I'm gonna punt on it :)